### DISCUSSION TITLE
Media library plugin CORS

### DISCUSSION THREAD

**Discussion by Badra00** [strapi/strapi#8057] [Legacy Topics]
Hello,

I would like to add cors on the media library plugin request.
How can I do this? (I have tried security.json and cors middleware)

Thanks

**Comment by derrickmehaffy** [comment]
The configuration structure for the middlewares has changed, see the following documentation: https://strapi.io/documentation/v3.x/concepts/middlewares.html#security-middlewares

**Reply by Badra00** [reply]
Hello, I have tried this solution, but it does nothing on the media plugin urls

**Reply by derrickmehaffy** [reply]
Can you describe what kind of errors you are seeing?

**Reply by Badra00** [reply]
Hello,
I'm adding this in the middleware.js file
```
module.exports = () => ({
  settings: {
    cors: {
      enabled: true,
      origin: ["*"]
    }
  },
});
```
But I do not see the header added to the response:
```
< HTTP/1.1 200 OK
< Vary: Origin
< Strict-Transport-Security: max-age=31536000; includeSubDomains
< X-Frame-Options: SAMEORIGIN
< Accept-Ranges: bytes
< Content-Length: 352948
< Last-Modified: Fri, 25 Sep 2020 21:20:55 GMT
< Cache-Control: max-age=0
< Content-Type: application/octet-stream
< X-Powered-By: Strapi <strapi.io>
< X-Response-Time: 1ms
< Date: Tue, 29 Sep 2020 10:50:34 GMT
< Connection: keep-alive
```

**Reply by MattieBelt** [reply]
Can u share an example of the request URL?

**Reply by Badra00** [reply]
curl -v http://localhost:1337/uploads/:id
